abci/example/counter/counter.go

```diff
@@ -107,11 +109,11 @@ func (app *Application) Query(reqQuery types.RequestQuery) types.ResponseQuery {
 }
 
 func (app *Application) EndBlock(reqEndBlock types.RequestEndBlock) types.ResponseEndBlock {
-	if app.HasChainLocks {
-		app.CurrentChainLockHeight = uint32(int32(app.CurrentChainLockHeight) + app.ChainLockStep)
-		chainLock := types1.NewMockChainLock(app.CurrentChainLockHeight)
+	if app.HasCoreChainLocks {
+		app.CurrentCoreChainLockHeight = uint32(int32(app.CurrentCoreChainLockHeight) + app.CoreChainLockStep)
+		chainLock := types1.NewMockChainLock(app.CurrentCoreChainLockHeight)
 		return types.ResponseEndBlock{
-			NextChainLockUpdate: chainLock.ToProto(),
+			CoreChainLockUpdate: chainLock.ToProto(),
```

Comment:
I would like to keep the Next part here to differentiate a chain lock update and this. Which is not an update in consensus, but an update to the next chain lock that the node would propose.

Reply:
Yeah, but this interface from ABCI app side. I was thinking just to keep name consistent with `ValidatorUpdates` and `ConsensusParamUpdates`.